alicloud-test/tests/alicloud_vpc_flow_log/variables.tf

```
@@ -0,0 +1,63 @@
+
+variable "resource_name" {
+  type        = string
+  default     = "turbot-test-20200125-create-update"
+  description = "Name of the resource used throughout the test."
+}
+
+variable "alicloud_region" {
+  type        = string
+  default     = "us-east-1"
+  description = "Alicloud region used for the test."
+}
+
+provider "alicloud" {
+  region = var.alicloud_region
+}
+
+data "alicloud_caller_identity" "current" {}
+
+data "null_data_source" "resource" {
+  inputs = {
+    scope = "acs:::${data.alicloud_caller_identity.current.account_id}"
+  }
+}
+
+variable "log_store_name" {
+  default = "vpc-flow-log-for-vpc"
+}
+
+variable "project_name" {
+  default = "vpc-flow-log-for-vpc"
+}
+
+resource "alicloud_vpc" "named_test_resource" {
+  cidr_block = "192.168.0.0/24"
+  name       = var.resource_name
```

From alicloud provider `1.119.0`, `name` attribute is deprecated.
Use `vpc_name` instead.
```suggestion
  vpc_name       = var.resource_name
```